package.json

```diff
@@ -13,8 +13,8 @@
     "react-test-renderer": "16.4.1",
     "xo": "^0.21.1"
   },
-  "_main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
-  "main": "./index.js",
+  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
+  "_main": "./index.js",
```

Comment:
.DS_Store and package-lock.json need to be removed. We already have yarn.lock

The package.json change needs to be removed. I know that it sucks, the example should really be in a subfolder, so that we don't have this problem.